src/main/java/com/github/kornilova_l/flamegraph/plugin/server/converters/file/jmc/JMCToCFlamegraphConverter.java

```diff
@@ -0,0 +1,182 @@
+package com.github.kornilova_l.flamegraph.plugin.server.converters.file.jmc;
+
+import com.github.kornilova_l.flamegraph.plugin.server.converters.calltraces.flamegraph.StacksParser;
+import com.github.kornilova_l.flamegraph.plugin.server.converters.file.CFlamegraph;
+import com.github.kornilova_l.flamegraph.plugin.server.converters.file.ProfilerToCompressedFlamegraphConverter;
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.util.text.StringUtil;
+import com.intellij.util.PathUtil;
+import org.apache.commons.lang.SystemUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.ZipException;
+
+public class JMCToCFlamegraphConverter implements ProfilerToCompressedFlamegraphConverter {
```

Comment:
Let's modify `JMCConverter` because it's better to keep file history. We can remove deprecation if we want!

Reply:
👍
Will do.